source/main.ts

```diff
@@ -181,7 +185,7 @@ app.on('activate', function () {
  * Hook into the unhandledRejection-event to prevent nasty error messages when
  * a Promise is rejected somewhere.
  */
-process.on('unhandledRejection', (err) => {
+processInfo.on('unhandledRejection', (err) => {
```

Comment:
This doesn't work at all, I always wanted to do this properly, but haven't had the time :D